packages/iTwinUI-react/src/core/ColorPicker/ColorSwatch.tsx

```diff
@@ -17,7 +17,7 @@ export type ColorSwatchProps = {
    * Is color selected.
    */
   isActive?: boolean;
-} & Omit<React.ComponentPropsWithRef<'div'>, 'color'>;
+} & Omit<React.ComponentPropsWithoutRef<'div'>, 'color'>;
```

Comment:
Otherwise it takes `css` from somewhere (I guess storybook). Checked whether it still allows to pass `ref`

Reply:
https://github.com/iTwin/iTwinUI-react/runs/6424346603?check_suite_focus=true#step:8:17

Reply:
Somehow it fixed itself 🤷‍♂️